src/Symfony/EventListener/DeserializeListener.php

```diff
@@ -62,7 +62,7 @@ public function onKernelRequest(RequestEvent $event): void
             return;
         }
 
-        if (null === $operation->canDeserialize() && $operation instanceof HttpOperation) {
```

Comment:
$this->initializeOperation always returns a HttpOperation and it use native return type.

Same idea for all the following `$operation instanceof HttpOperation`